dataplane/windows/policysets/policysets.go

```diff
@@ -530,21 +542,27 @@ func (s *PolicySets) NewRule(isInbound bool, priority uint16) *hns.ACLPolicy {
 	}
 }
 
-// NewHostRule returns a new hns rule object scoped to the host. This is only
-// temporarily required for compatibility with RS3.
+// NewHostRule returns a new hns rule object scoped to the host.
 func (s *PolicySets) NewHostRule(isInbound bool) *hns.ACLPolicy {
 	direction := hns.Out
 	if isInbound {
 		direction = hns.In
 	}
 
+	priority := 0
```

Comment:
If you define this as `var priority uint16` then you wouldn't need the cast below